kinto_amo/__init__.py

```diff
@@ -15,14 +16,33 @@ def includeme(config):
     # Parse resources
     settings = config.get_settings()
     resources = OrderedDict()
-    resources['addons'] = parse_resource(
-        settings.get('amo.addons', DEFAULT_ADDONS))
-    resources['plugins'] = parse_resource(
-        settings.get('amo.plugins', DEFAULT_PLUGINS))
-    resources['gfx'] = parse_resource(
-        settings.get('amo.gfx', DEFAULT_GFX))
-    resources['certificates'] = parse_resource(
-        settings.get('amo.certificates', DEFAULT_CERTIFICATES))
+
+    # Configure default settings
+    resources['blocklist'] = OrderedDict()
```

Comment:
Can we comment here why we use an ordereddict? (even if it says it's for reproducible tests)